src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java

```diff
@@ -48,16 +43,11 @@
 	
 	private final ClientService clientService;
 	
-	@Autowired
-	private Dhis2TrackCaptureConnector dhis2TrackCaptureConnector;
```

Comment:
This seems to be DHIS specific while remove it?

Reply:
This field also has a **private** access privileges but is unused anywhere in this class hence it is not required either